index.js

```diff
@@ -521,6 +544,11 @@
     return typeof other === typeof this && other.valueOf() === this.valueOf();
   }
 
+  //  Boolean$prototype$lte :: Boolean ~> Boolean -> Boolean
+  function Boolean$prototype$lte(other) {
+    return typeof other === typeof this && this.valueOf() <= other.valueOf();
+  }
+
```

Comment:
In Sanctuary proper we decided not to work with Boolean, Number, and String objects. Since sanctuary-type-classes is essentially the standard library for Fantasy Land, we should not be so opinionated. As a result, we must accept that there are *four* Boolean values: `false`, `true`, `new Boolean(false)`, and `new Boolean(true)`. How should these values be ordered?